nuxeo-mqueues-core/src/main/java/org/nuxeo/ecm/platform/importer/mqueues/mqueues/chronicle/ChronicleMQueue.java

```diff
@@ -207,5 +261,79 @@ private int findNbQueues(File basePath) {
         return ret;
     }
 
+    private RollCycle getRollCycle(String retentionDuration) {
+        String rollingPeriod = retentionDuration.substring(retentionDuration.length() - 1);
+        RollCycle rollCycle;
+        switch (rollingPeriod) {
+        case SECOND_ROLLING_PERIOD:
+            rollCycle = RollCycles.TEST_SECONDLY;
```

Comment:
Should we really use `TEST_SECONDLY` though? It's commented as _only good for testing_ in its definition, and there is TEST in the name which is suspicious. @bdelbosc ?